x-pack/plugins/infra/public/pages/metrics/metric_detail/asset_detail_page.tsx

```diff
@@ -70,7 +70,7 @@ export const AssetDetailPage = () => {
     return queryParams.get('assetName') ?? undefined;
   }, [search]);
 
-  const { timeRange, setTimeRange } = useMetricsTimeContext();
+  const { timeRange } = useMetricsTimeContext();
```

Comment:
There are now 2 sources of truth for getting the `timeRange` from the URL. This  one, when the page loads, and the `assetDetails` URL variable. I think it's OK for now, but we'll have to stop using this context in this view in the future.

Reply:
Agree! I even tried to unify it a bit and use `assetDetails` from the URL everywhere but stumbled over some places that would need a refactoring in order for this to work (specifically the logic to generate asset details URLs in the hosts table and "Open as page" in flyout). I was not courageous enough yet to refactor those 😅

Reply:
yeah. better to do it in a separate PR.